src/main/java/net/snowflake/client/core/SessionUtil.java

```diff
@@ -72,12 +72,11 @@ public class SessionUtil {
   public static final String SF_QUERY_SESSION_DELETE = "delete";
 
   // Headers
-  public static final String SF_HEADER_AUTHORIZATION = HttpHeaders.AUTHORIZATION;
+  @Deprecated
+  public static final String SF_HEADER_AUTHORIZATION = SFSession.SF_HEADER_AUTHORIZATION;
```

Comment:
I'm wondering if SFSession is the best place to place request headers, but probably we have to live with it.

Reply:
Yes, we have to - eventually we should make those properties private in driver